models/dao/KnexConfig.js

```diff
@@ -12,6 +12,13 @@ module.exports = require('../Model').extend({
       type: Boolean,
       description: 'Run Knex in debug mode',
       default: false
+    },
+    migrations: {
+      type: Object,
+      description: 'Migrations config object',
+      default: {
```

Comment:
Please just add a `TODO` here to remind us to revisit this code after `fashion-model` propagates mixins.

Reply:
Went ahead and adopted new defaults changes, even though they aren't merged yet. We'll need a new version published before relying on the defaulting behavior in the other repos.